core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1039,10 +1039,10 @@ class HttpGoogleServicesDAO(
       .build()
   }
 
-  def toProxyFromUser(rawlsUser: RawlsUser) = toProxyFromUserSubjectId(rawlsUser.userSubjectId.value)
-  def toProxyFromUser(userInfo: UserInfo) = toProxyFromUserSubjectId(userInfo.userSubjectId)
-  def toProxyFromUser(subjectId: RawlsUserSubjectId) = toProxyFromUserSubjectId(subjectId.value)
-  def toProxyFromUserSubjectId(subjectId: String) = s"PROXY_${subjectId}@${appsDomain}"
+  def toProxyFromUser(rawlsUser: RawlsUser) = toProxyFromUserSubjectIdString(rawlsUser.userSubjectId.value)
+  def toProxyFromUser(userInfo: UserInfo) = toProxyFromUserSubjectIdString(userInfo.userSubjectId.value)
+  def toProxyFromUser(subjectId: RawlsUserSubjectId) = toProxyFromUserSubjectIdString(subjectId.value)
+  def toProxyFromUserSubjectIdString(subjectId: String) = s"PROXY_${subjectId}@${appsDomain}"
```

Comment:
There appears to be a form of `toProxyFromUser` above that does. What I wonder is if `toProxyFromUserSubjectIdString` can simply be inlined into that form and have the `RawlsUser` and `UserInfo` forms delegate to it.